src/src/PluginStructs/P093_data_struct.h

```diff
@@ -182,6 +185,7 @@ struct P093_data_struct : public PluginTaskData_base {
              wideVane != rhs.wideVane ||
              iSee != rhs.iSee ||
              roomTemperature != rhs.roomTemperature ||
+             remoteTemperature != rhs.remoteTemperature ||
```

Comment:
Here you can use a specific, more accurate, float compare function:
`!essentiallyEqual(remoteTemperature, rhs.remoteTemperature)`
(can also be applied to the other 2 float values `temperature` and `roomTemperature`)